### PR TITLE
add package: timezone + spelling fix for crone package

### PR DESCRIPTION
**Pull request by wrzonki**
- added package: **timezone**
- fixed name of package that I misspelled: "**cron**" insted of "**crone**" from [this pr](https://github.com/denoland/deno_website2/pull/475?fbclid=IwAR3UlfkbRZkTAHX6YW8a6Ndvg5xhNXm20XfFUmDNgiw8bRAXz40aKtFwxXY#event-3338901628)

### CODE REVIEW

File: database.json [hunk 1]
```diff
@@ -228,11 +228,11 @@
...
     "desc": "Configurable talking cow for Deno"
   },
-  "crone": {
+  "cron": {
     "type": "github",
     "owner": "wrzonki",
-    "repo": "deno-crone",
-    "desc": "Simple crone scheduler for Deno"
+    "repo": "deno-cron",
+    "desc": "Simple cron scheduler for Deno"
```
Comment (parzhitsky) [inline database.json:235]: @ry [might not be happy](https://github.com/denoland/deno_website2/pull/486#discussion_r425933047) about this change 😄

Comment (wrzonki) [reply database.json:235]: It was today at 2am - my bad.
Should I remove it, or wait for @ry to decide?

**Comment by parzhitsky** [comment]
I’d split it into two PRs

File: database.json (hunk 1 above)
Comment (parzhitsky) [reply database.json:235]: I’d split it into two PRs

**Pull request merged**